src/js/lightbeamVR.js

```diff
@@ -0,0 +1,36 @@
+// eslint-disable-next-line no-unused-vars
+const lightbeamVR = {
+  init() {
+    this.loadLightbeamVR();
+  },
+
+  loadLightbeamVR() {
+    const vrView = document.getElementById('vr-view-button');
```

Comment:
can you create this button dynamically in JS, if it doesn’t exist? or throw a warning if it doesn’t exist?